### DISCUSSION TITLE
DisplayModel cannot be changed in Tasmota Display 14.0.0

### DISCUSSION THREAD

**Discussion by Cassilo** [arendst/Tasmota#21522] [Q&A]
In Tasmota Display 14.0.0 i can not change the DisplayModel. 
Whenever you try to change the display model, it is set back to 11.

**Comment by Jason2866** [comment]
See release notes of v14 for changes regarding Displays.

**Reply by sfromis** [reply]
And see https://tasmota.github.io/docs/Universal-Display-Driver/ for migration
